EventInfoBinder NPE

```
java.lang.IllegalStateException: Fatal Exception thrown on Scheduler.Worker thread.
	at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:62)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5343)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.setVisibility(int)' on a null object reference
	at com.thebluealliance.androidclient.binders.EventInfoBinder.lambda$onEvent$3(EventInfoBinder.java:324)
	at com.thebluealliance.androidclient.binders.EventInfoBinder.access$lambda$3(EventInfoBinder.java)
	at com.thebluealliance.androidclient.binders.EventInfoBinder$$Lambda$4.call(Unknown Source)
	at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:55)
	... 8 more
```

--- a/android/src/main/java/com/thebluealliance/androidclient/binders/EventInfoBinder.java
+++ b/android/src/main/java/com/thebluealliance/androidclient/binders/EventInfoBinder.java
@@ -316,6 +316,9 @@ public class EventInfoBinder extends AbstractDataBinder<EventInfoBinder.Model> {
     @Subscribe
     public void onEventRankingsUpdated(EventRankingsEvent event) {
         AndroidSchedulers.mainThread().createWorker().schedule(() -> {
+            if (topTeamsContainer == null || topTeams == null) {
+                return;
+            }
             topTeamsContainer.setVisibility(View.VISIBLE);
             topTeamsContainer.setOnClickListener(mInfoClickListener);
             topTeams.setText(Html.fromHtml(event.getRankString()));
@@ -326,6 +329,9 @@ public class EventInfoBinder extends AbstractDataBinder<EventInfoBinder.Model> {
     @Subscribe
     public void onEventStatsUpdated(EventStatsEvent event) {
         AndroidSchedulers.mainThread().createWorker().schedule(() -> {
+            if (topOprsContainer == null || topOprs == null) {
+                return;
+            }
             topOprsContainer.setVisibility(View.VISIBLE);
             topOprsContainer.setOnClickListener(mInfoClickListener);
             topOprs.setText(Html.fromHtml(event.getStatString()));

--- a/android/src/main/java/com/thebluealliance/androidclient/helpers/WebcastHelper.java
+++ b/android/src/main/java/com/thebluealliance/androidclient/helpers/WebcastHelper.java
@@ -1,12 +1,13 @@
 package com.thebluealliance.androidclient.helpers;
 
+import com.google.gson.JsonObject;
+
+import com.thebluealliance.androidclient.R;
+import com.thebluealliance.androidclient.types.WebcastType;
+
 import android.content.Context;
 import android.content.Intent;
 import android.net.Uri;
-
-import com.google.gson.JsonObject;
-import com.thebluealliance.androidclient.R;
-import com.thebluealliance.androidclient.types.WebcastType;
 
 public class WebcastHelper {
 
@@ -32,6 +33,10 @@ public class WebcastHelper {
     }
 
     public static Intent getIntentForWebcast(Context context, String eventKey, WebcastType type, JsonObject params, int number) {
+        if (type != WebcastType.IFRAME && !params.has("channel")) {
+            return getGamedayIntent(context, eventKey, number);
+        }
+
         switch (type) {
             case YOUTUBE:
                 return getWebIntentForUrl(context.getString(R.string.webcast_youtube_embed_pattern, params.get("channel").getAsString()));
@@ -40,6 +45,9 @@ public class WebcastHelper {
             case USTREAM:
                 return getWebIntentForUrl(context.getString(R.string.webcast_ustream_embed_pattern, params.get("channel").getAsString()));
             case LIVESTREAM:
+                if (!params.has("file")) {
+                    return getGamedayIntent(context, eventKey, number);
+                }
                 return getWebIntentForUrl(context.getString(R.string.webcast_livestream_embed_pattern, params.get("channel").getAsString(), params.get("file").getAsString()));
             case HTML5:
                 return new Intent(Intent.ACTION_VIEW).setDataAndType(Uri.parse(params.get("channel").getAsString()), "video/*");
@@ -47,8 +55,12 @@ public class WebcastHelper {
                 return new Intent(Intent.ACTION_VIEW).setDataAndType(Uri.parse(context.getString(R.string.webcast_stemtv_embed_pattern, params.get("channel").getAsString())), "video/*");
             default:
             case IFRAME:
-                return getWebIntentForUrl(context.getString(R.string.webcast_gameday_pattern, eventKey, number));
+                return getGamedayIntent(context, eventKey, number);
         }
+    }
+
+    public static Intent getGamedayIntent(Context context, String eventKey, int number) {
+        return getWebIntentForUrl(context.getString(R.string.webcast_gameday_pattern, eventKey, number));
     }
 
     public static Intent getWebIntentForUrl(String url) {

